_layouts/page_submenu.html

```diff
@@ -25,6 +25,9 @@
   <aside class="page-sidebar">
     <ul class="sidebar-nav">
       {% for link in page.sidebar %}
+      {% if link.break %}
```

Comment:
I just haven't used `br` in a decade because pure css seems cleaner but maybe it's not that important right now.